zcutil/tnbox/tnbox.py

```diff
@@ -0,0 +1,99 @@
+# /************************************************************************
```

Comment:
Should have #!

Should be executable?

Reply:
Can't remember if all the platforms use `#!/bin/env/python3` or `#!/bin/env/python` for python3 now. IIRC, macOS was the only picky one

Reply:
Oof, I meant : `#!/usr/bin/env python3`